test/Microsoft.AspNet.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -311,5 +310,17 @@ public class HttpsConnectionFilterTests
 #endif
             }
         }
+
+        private static string GetTestCertificatePath()
+        {
+            if (Directory.GetCurrentDirectory().Contains(".testPublish"))
```

Comment:
I'll use `"content": [ "TestResources/testCert.pfx" ]`, similar to what I did to include `hosting.json` in the sample apps. That way I don't have to go around looking for it at runtime.